crates/web-plugins/didcomm-messaging/protocol-plugins/routing/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "forward"
```

Comment:
Shouldn't this name match the folder's name?

Reply:
It was an error, I renamed the folder but forgot to change the package name as well.